### DISCUSSION TITLE
some suggestion

### DISCUSSION THREAD

**Discussion by solitudealma** [dunst-project/dunst#1387] [Ideas]
As the calendar notice in the readme, whether the rule matching setting can be implemented in subsequent updates is equivalent to an independent configuration, or in other words, dunst can implement functions such as dynamic switching configuration. Although the rule can complete some work, it only changes the color, etc., and the size and shape are fixed.

**Comment by bynect** [comment]
we recently added hot reloading configuration, maybe something of what you want can be done with that?

**Reply by solitudealma** [reply]
ok, i take a try
